Framework/DataHandling/src/LoadMask.cpp

```diff
@@ -504,344 +675,55 @@ void LoadMask::parseXML() {
     } else if (pNode->nodeName().compare("ids") == 0) {
       // Node "ids"
       if (ingroup) {
-        this->parseSpectrumNos(value, m_MaskSpecID);
+        parseRangeText(value, singleSp, pairSp);
+        // this->parseSpectrumNos(value, m_maskSpecID);
```

Comment:
If this is unused could we remove it? If not could you comment why not so it doesn't get erroneously removed